<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrainWindow</class>
 <widget class="QMainWindow" name="TrainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Train dialogue</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>110</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Speed</string>
    </property>
   </widget>
   <widget class="QComboBox" name="door_lock">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>111</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>No</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Yes</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Doors locked</string>
    </property>
   </widget>
   <widget class="QComboBox" name="door_open">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>110</y>
      <width>111</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>No</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Yes</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="station">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>170</y>
      <width>111</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>No</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Yes</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Doors open</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_butt">
    <property name="geometry">
     <rect>
      <x>43</x>
      <y>391</y>
      <width>291</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>PPM</string>
    </property>
   </widget>
   <widget class="QComboBox" name="lock_out">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>310</y>
      <width>111</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>No</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Yes</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Speed</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Emergency brake</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Doors locked</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Doors free</string>
    </property>
   </widget>
   <widget class="QComboBox" name="open_out">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>340</y>
      <width>111</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>No</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Yes</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="PPM">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>111</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>&lt;180</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>180</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>270</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>420</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>220</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Outputs</string>
    </property>
   </widget>
   <widget class="QComboBox" name="brake_out">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>280</y>
      <width>111</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>No</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Yes</string>
     </property>
    </item>
   </widget>
   <widget class="QSlider" name="speed">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>50</y>
      <width>21</width>
      <height>141</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ClosedHandCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="maximum">
     <number>55</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="invertedAppearance">
     <bool>true</bool>
    </property>
    <property name="tickInterval">
     <number>5</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="speed_out">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>250</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>45</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>350</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Doors open</string>
    </property>
   </widget>
   <widget class="QComboBox" name="door_free">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>140</y>
      <width>111</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>No</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Yes</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="speed_val">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>110</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Inputs</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>At station</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>382</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>speed</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>speed_val</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>320</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
